tools/roslaunch/src/roslaunch/loader.py

```diff
@@ -406,11 +403,9 @@ def load_rosparam(self, context, ros_config, cmd, param, file_, text, verbose=Tr
                 text = subst_function(text)
             # parse YAML text
             # - lazy import: we have to import rosparam in oder to to configure the YAML constructors
-            global rosparam
-            if rosparam is None:
-                import rosparam
+            import rosparam
```

Comment:
Why has this import logic been changed? It looks unrelated to the patch.

Reply:
You're right, it's unrelated to the functional change in this patch.

The reason I touched it was that I was checking if/where `rosparam` was imported within `roslaunch` globally and finding the global `rosparam` variable confused me. Then I googled for this construction and came across https://stackoverflow.com/a/4177780/177643, where it's pointed out that python already caches imports, so this optimization would not really help performance.
Aside of that, it pollutes the interface of this python file, by exposing `rosparam` which might be `None` or the loaded `rosparam` module, depending on if this function has been called or not. Since this is the only use of rosparam within this module, I thought it would be a good idea to clean up.

It's fine for me to roll this back, you can decide.

Reply:
Please remove the change from this PR since the title won't let you see that this is part of the patch. Please feel free tp create a separate PR with this change.